Source/Service/RoomInvitationRedeemer.swift

```diff
@@ -93,10 +93,17 @@ enum RoomInvitationRedeemer {
         return true
     }
     
+    // Redeems token with a given `MultiserverAddress`
+    static func redeem(address: MultiserverAddress, token: String, in controller: AppController, bot: Bot) async {
+        if let inviteUrl = URL(string: "https://\(address.host)/join?token=\(token)") {
```

Comment:
I think this function should work with planetary.name - it is accepting the magic token now. But the call is a little off. I was able to join the room by replacing the body of the function with this:
`try! await RoomInvitationRedeemer.redeem(token: token, at: address.host, bot: bot)`

(obviously we will not want to force the `try` in production)
Note that if any user on the device has already joined planetary.name you will get an error that you have already joined the room due to #898.

After joining the room I wasn't able to set an alias. Not sure why, I didn't look into too much. But I'm happy to if you'd like.